models/profile.js

```diff
@@ -0,0 +1,28 @@
+// var mongoose = require('mongoose'),
+// const Schema = mongoose.Schema;
+
+// var CampsiteSchema = new Schema({
+//   description: String
+// });
+
+// var Campsite = mongoose.model('Campsite', CampsiteSchema);
+
+// module.exports = Campsite;
+
+var mongoose = require('mongoose'),
+Schema = mongoose.Schema;
+
+const ProfileSchema = new Schema ({
```

Comment:
It seems like this is data about a motorcycle; why is it in a file called Profile?